### PR TITLE
Small rework of background api & consistency of it 

### PR DESCRIPTION
**Pull request by louis1706**

### CODE REVIEW

File: Exiled.API/Features/Lift.cs [hunk 1]
```diff
@@ -390,7 +390,7 @@ public void Lock(DoorLockReason lockReason = DoorLockReason.Isolation)
         public void Lock(float duration, DoorLockReason lockReason = DoorLockReason.Isolation)
         {
             Status = ElevatorSequence.DoorClosing;
-            ChangeLock(lockReason);
+            Lock(duration, lockReason);
```
Comment (VALERA771) [inline Exiled.API/Features/Lift.cs:393]: Calling same method?

File: Exiled.Events/Patches/Generic/TargetOffset.cs [hunk 2]
```diff
@@ -7,6 +7,7 @@
 
 namespace Exiled.Events.Patches.Generic
 {
+#pragma warning disable IDE0060 // Supprimer le paramètre inutilisé
```
Comment (VALERA771) [inline Exiled.Events/Patches/Generic/TargetOffset.cs:10]: What this warning about? Seems to be IDE specific so idk if we really need it

File: Exiled.API/Features/Lift.cs (hunk 1 above)
Comment (louis1706) [reply Exiled.API/Features/Lift.cs:393]: Yup

File: Exiled.Events/Patches/Generic/TargetOffset.cs (hunk 2 above)
Comment (IkaOverride) [reply Exiled.Events/Patches/Generic/TargetOffset.cs:10]: I'm kinda late but it says "Remove unused parameter". so it removes parameters being flagged when they aren't used and they do not start with the _ symbol

Comment (NaoUnderscore) [inline Exiled.Events/Patches/Generic/TargetOffset.cs:10]: I don't want french in Exiled's code 👎🏼

**Review by NaoUnderscore — CHANGES_REQUESTED** [review]

File: Exiled.Events/Patches/Generic/TargetOffset.cs (hunk 2 above)
Comment (louis1706) [inline Exiled.Events/Patches/Generic/TargetOffset.cs:10]:
```suggestion
```

**Review by NaoUnderscore — APPROVED** [review]

**Pull request merged**